scripts/score_real_collection.py

```diff
@@ -0,0 +1,40 @@
+from open_rarity import OpenRarityScorer
+from open_rarity.resolver.opensea_api_helpers import (
+    get_collection_from_opensea,
+)
+from open_rarity.resolver.testset_resolver import extract_rank
+
+if __name__ == "__main__":
+    """This script fetches bored ape yacht club collection and token metadata
+    from the Opensea API via opensea_api_helpers and scores the
+    collection via OpenRarity scorer.
+    """
+    scorer = OpenRarityScorer()
+    slug = "boredapeyachtclub"
+
+    # Get collection
+    collection = get_collection_from_opensea(slug)
+    print(f"Created collection with {collection.token_total_supply} tokens")
+
+    # Score the entire collection
+    token_scores = scorer.score_collection(collection=collection)
+    print(
+        f"Calculated {len(token_scores)} token scores for collection: {slug}"
+    )
+
+    # Print out scores
+    for i, token_score in enumerate(token_scores):
+        print(f"\tToken {collection.tokens[i]} has score: {token_score}")
+
+    # Convert scores to rank
+    token_id_to_rank_score = extract_rank(
```

Comment:
nit: this rank assignment doesn't have same score per tokens handling logic

Reply:
I may missed it, but where do we sort tokens?

Reply:
yep we will resolve this in a follow up PR to have a `Ranker` interface